### DISCUSSION TITLE
Reset 2FA for admin

### DISCUSSION THREAD

**Discussion by dima-online** [requarks/wiki#5254] [Help / Questions] [answered]
Hi
I broke my phone's screen so I can't transfer Google authenticator settings.
I can't log in as an administrator to the wiki because my account is set to 2fa Google authenticator.
I know my account password.

How do I disable 2FA for my account?

**Comment by NGPixel** [comment, marked as answer]
Assuming you're running postgres in docker (replace YOUR-EMAIL):

```
docker exec db psql -U wiki -d wiki -c "UPDATE users SET tfaIsActive = false WHERE email = 'YOUR-EMAIL';"
```

**Reply by vosgames** [reply]
Hi, I got the same problem but I am running wiki js on a docker. Please help me I don’t wanna lose all my data

**Reply by Philipinho** [reply]
@vosgames have you tried the command above?

**Reply by vosgames** [reply]
@Philipinho yes I tried it but I am using the database postgres. Couldn’t get it to work.

**Reply by Philipinho** [reply]
@vosgames did you replace "YOUR-EMAIL" with your wikijs registered email?
If yes, when you ran the command, what was the output?

**Reply by vosgames** [reply]
@Philipinho 
This is the error. I am pretty sure its the right email. Because when i try to log-in trough GUI i get this pop up.
![image](https://user-images.githubusercontent.com/39599164/222982351-f4a566b4-c7ca-4075-857f-f28efcf29274.png)


When i try to run the command above here it gives this messages (below here the command)
docker exec "containerID" psql -U "username postgres_user" -d "databasename"-c "UPDATE users SET tfaIsActive = false WHERE email = "email adres"

Then the error underneath happends:
 
_ERROR:  column "tfaisactive" of relation "users" does not exist
LINE 1: UPDATE users SET tfaIsActive = false WHERE email =_

**Reply by Philipinho** [reply]
@vosgames I know why now. You have to escape it `\"tfaIsActive\"`:
`docker exec db psql -U wiki -d wiki -c "UPDATE users SET \"tfaIsActive\" = false WHERE email = 'YOUR-EMAIL';"`

**Reply by vosgames** [reply]
@Philipinho You are amazing! thanks that worked :)
